As2 v6 template updates mc (#970)
mulesoft/docs-connectors: as2/6.0/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc

> = Upgrade AS2 - Mule 4

Reviewer: ```suggestion
= Upgrade AS2 Connector to Version 6.x - Mule 4
```

Reviewer: Not sure if you can enter 6.0. Per previous edits in this doc the number 6.0.0 was changed to 6.x

|5.x |6.x
|===

== Changes in This Release 

AS2 Connector 6.0 contains the following changes:

> * Digitally signed copies of outgoing and received messages and their corresponding receipts can be saved for non-repudiation purposes as long as they are configured and enabled on AS2 Connector operations and sources.

Reviewer: ```suggestion
* Digitally signed copies of outgoing and received messages and their corresponding receipts can be saved for nonrepudiation purposes as long as they are configured and enabled on AS2 Connector operations and sources.
```

> * The new Mule source `Non repudiation listener` must be user-configured to persist the non-repudiation information provided by any of the AS2 Connector

Reviewer: ```suggestion
* The new source `Non repudiation listener` must be user configured to persist the non-repudiation information provided by any of the AS2 Connector
```

Reviewer: I haven't found examples of user-configured, but user-defined exists. So feel free to take it or leave it.
Also non-repudiation should be nonrepudiation per Style guide, but in Google searches and throughout the whole doc it's called non-repudiation, so I think you can leave it.

operations and sources via a user-defined persistence strategy.
* Certain attribute object fields are renamed and deprecated fields are removed. +
These changes are outlined in <<upgrade-steps,Upgrade Steps>>.

== Requirements

Ensure that you meet the following requirements before you migrate:

* Mule 4.3.0 or later
* Java 8

[[upgrade-steps]]
== Upgrade Steps

Follow these steps to perform the upgrade to AS2 version 6.x:

> . Download AS2 connector 6.x.x from Exchange. 

Reviewer: ```suggestion
. Download AS2 Connector 6.x.x from Exchange. 
```
